invokeai/backend/model_manager/probe.py

```diff
@@ -554,12 +560,21 @@ class LoRACheckpointProbe(CheckpointProbeBase):
     """Class for LoRA checkpoints."""
 
     def get_format(self) -> ModelFormat:
-        return ModelFormat("lycoris")
+        if is_state_dict_likely_in_flux_diffusers_format(self.checkpoint):
+            # TODO(ryand): This is an unusual case. In other places throughout the codebase, we treat
+            # ModelFormat.Diffusers as meaning that the model is in a directory. In this case, the model is a single
+            # file, but the weight keys are in the diffusers format.
+            return ModelFormat.Diffusers
```

Comment:
Question for reviewers: Any concerns with this?

Reply:
After looking through the flow I don't believe there are significant concerns here. LoRALoader._get_model_path does treat this format as a declaration of the model being a directory, but it doesn't seem to have any issues with it being a checkpoint file instead.